### ISSUE TITLE
Image Alt Text

### ISSUE THREAD

**Issue by ApothecaryOfLight** [ApothecaryOfLight/Portfolio#24]
Images should have alt text for both accessibility and robustness (in case the image fails to display for whatever reason).
